### ISSUE TITLE
All staging order crashing

### ISSUE THREAD

**Issue by jordansorensen** [printdotio/printio-ios-sdk#490]
I currently can't place any orders in staging - when I go to place my order, the images start uploading to S3, and immediately after the last image loads the app crashes with the following errors:

> 2015-11-13 10:33:51.620 (app)[487:125694] Can't endBackgroundTask: no background task exists with identifier 27, or it may have already been ended. Break in UIApplicationEndBackgroundTaskError() to debug.
> 2015-11-13 10:33:52.827 (app)[487:121046] Problem deleting the peer public key from keychain, OSStatus == 4294933278.
> 2015-11-13 10:33:52.831 (app)[487:121046] Problem adding the public key, OSStatus == 4294933278.
> 2015-11-13 10:33:52.833 (app)[487:121046] Problem acquiring reference to the public key, OSStatus == 4294933278.

**Comment by perisicboro** [comment]
@jordansorensen We can't recreate this issue, orders are working well on our test devices. This log is not helping too much, please try to catch another with crash, and send me sdk initialization code on boro@makeable.com

**Comment by jordansorensen** [comment]
This happens for me >95% of the time, seemingly regardless of the settings I use. This morning I've been testing with no customization at all:
>PrintIO* printIO = [[PrintIO alloc] initWithViewController:self environment:PRINTIO_STAGING productionRecipeId:pRecipeId stagingRecipeId:sRecipeId];
>[printIO open];

1. Select a product (seems to be any product; I've confirmed this behavior with prints, acrylic prints, puzzles, and magnets).
2. Proceed through selecting options, depending on which product you've selected.
3. Select photos. I've confirmed this crash both using "Phone" as my photo source, and with setImages + disablePhotoSourcesWhenImagesArePassedIn:YES. I've confirmed the behavior with the number of photos between from 1-100.
4. Add to cart, check out. Enter shipping+billing info. I've been using "Card" as my payment source; I haven't tested PayPal.
5. Place order. Images upload, then screen changes to "Finishing Upload...", then crash described above occurs.

I can send you my staging recipe ID separately via email if that's helpful.
